partials/home-splash.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$home = get_page_by_path('home');
+$splash_text = get_post_meta($home->ID,'_igv_splash_text', true);
```

Comment:
this depends on a `home` page existing and thus errors when the page doesn't exist.